auth-web/src/components/auth/common/PaymentMethods.vue

```diff
@@ -310,18 +315,22 @@ export default defineComponent({
       const paymentMethods = []
       if (props.currentOrgType) {
         const paymentTypes = paymentsPerAccountType[props.currentOrgType]
-        paymentTypes.forEach((paymentType) => {
-          if (PAYMENT_METHODS[paymentType]) {
-            paymentMethods.push(PAYMENT_METHODS[paymentType])
-          }
-        })
+        if (paymentTypes) {
```

Comment:
(paymentTypes || []).forEach() might be easier

Reply:
update to: paymentTypes?.forEach()